src/weapons/scout/AssaultRifle.java

```diff
@@ -308,7 +308,7 @@ else if (selectedOverclock == 5) {
 	public StatsRow[] getStats() {
 		StatsRow[] toReturn = new StatsRow[12];
 		
-		boolean directDamageModified = selectedTier2 == 0 || selectedTier3 == 1 || selectedOverclock == 2 || selectedOverclock == 5 || selectedOverclock == 6;
+		boolean directDamageModified = selectedTier2 == 0 || selectedTier3 == 1 || selectedOverclock == 2 || selectedOverclock == 5;
```

Comment:
Good catch on the StatsRow for the DirectDamage penalty no longer applying. In the PR, I don't see the corresponding change to indicate that the StatsRow for CarriedAmmo should change when Electrifying Reload is equipped.